core/geometry/src/test/geometry3d/OrderedRotationAngles.test.ts

```diff
@@ -603,7 +697,8 @@ describe("OrderedRotationAngles", () => {
     const matrix = Matrix3d.createIdentity();
     const angles = OrderedRotationAngles.createFromMatrix3d(matrix, AxisOrder.XYZ); // order doesn't matter
     const expectedAngles = OrderedRotationAngles.createRadians(0, 0, 0, AxisOrder.XYZ);
-    testEqualOrderedRotationAngles(ck, angles, expectedAngles);
+    if (angles !== undefined)
```

Comment:
Same as prev comment: add `ck.testDefined(angles)` to the if statement since we expect `createFromMatrix3d` to succeed on these inputs.